pom.xml

```diff
@@ -23,12 +23,12 @@ under the License.
   <parent>
     <groupId>org.apache.maven.plugins</groupId>
     <artifactId>maven-plugins</artifactId>
-    <version>42</version>
+    <version>43</version>
```

Comment:
These updates should be separate PRs. Might already be done if you merge to master.